src/inference/dev_api/openvino/runtime/iasync_infer_request.hpp

```diff
@@ -37,8 +37,8 @@ namespace ov {
 class OPENVINO_RUNTIME_API IAsyncInferRequest : public IInferRequest {
 public:
     IAsyncInferRequest(const std::shared_ptr<IInferRequest>& request,
-                       const InferenceEngine::ITaskExecutor::Ptr& task_executor,
-                       const InferenceEngine::ITaskExecutor::Ptr& callback_executor);
+                       const std::shared_ptr<ov::threading::ITaskExecutor>& task_executor,
+                       const std::shared_ptr<ov::threading::ITaskExecutor>& callback_executor);
```

Comment:
why don't you like to use `::Ptr`?

Reply:
I prefer to have one style for out classes. We don't introduce `::Ptr` in other structures.